File: Tareas_FdP/Tarea02vscode/Ejercicio10.py
import math

print("Ingrese el radio de la esfera: ")
radio = float(input())

area = 4 * math.pi * (radio ** 2)
volumen = (4 * math.pi * (radio ** 3)) / 3.0

print("El área de la esfera es: ", area)
print("El volumen de la esfera es: ", volumen)

File: Tareas_FdP/Tarea02vscode/Ejercicio10.cs
using System;

class Esfera {
    static void Main() {
        double radio, area, volumen;

        Console.WriteLine("Ingrese el radio de la esfera: ");
        radio = Convert.ToDouble(Console.ReadLine());

        area = 4 * Math.PI * (radio * radio);
        volumen = (4 * Math.PI * (radio * radio * radio)) / 3.0;

        Console.WriteLine("El área de la esfera es: " + area);
        Console.WriteLine("El volumen de la esfera es: " + volumen);
    }
}